### PR TITLE
add DMR code blocks

### PR DESCRIPTION
**Pull request by ericnost**
This next draft of the Watershed notebook fixes some issues in mapping watersheds and also adds metrics for evaluating pollutant discharges.

### CODE REVIEW

**Comment by ericnost** [comment]
Test run: https://colab.research.google.com/drive/1dK85SpAR9Dvlq0qcvJCO1CMx8hdvvz2K?usp=sharing

**Comment by shansen5** [comment]
I've been thinking that in the top violators graph if there are more than 20 with all quarters in violation we should let the graph grow larger.

**Comment by ericnost** [comment]
@shansen5 I have thought the same! Especially with Clean Water Act violations, we are seeing many facilities, much more than 20, that are always in violation. I wonder if there is a way we could say something like, show all the facilities with at least 3 quarters of violations, to a max of 50 facilities (or some other reasonable threshold)?

File: ECHO_Watershed.ipynb [hunk 1]
```diff
@@ -245,18 +257,17 @@
       },
       "source": [
         "huc = gdf.at[selection.value, \"huc8\"]\n",
-        "huc = float(huc)\n",
+        "huc = float(huc) #string or integer?\n",
```
Comment (Frijol) [inline ECHO_Watershed.ipynb]: should this comment still be here?

File: ECHO_Watershed.ipynb [hunk 2]
```diff
@@ -344,7 +355,7 @@
         "id": "e2F5WUKZbBM6"
       },
       "source": [
-        "Get the data from the SBU database. **THIS WON'T WORK UNTIL WE EDIT SOME PARAMETERS IN ECHO_MODULES....STAY TUNED!!!** "
+        "Get the data from the Stony Brook University database. "
```
Comment (Frijol) [inline ECHO_Watershed.ipynb:448]:
```suggestion
        "Get the data from the Stony Brook University database."
```

**Review by Frijol — APPROVED** [review]
Looks great, well-commented!

**Comment by Frijol** [comment]
Is this PR ready to merge? If not, what is pending?

**Comment by ericnost** [comment]
There are still some loose ends at the end of the notebook. I will try to clean up, commit, and merge.

**Comment by Frijol** [comment]
No rush, just wanted to make sure we weren't leaving a PR sitting

**Comment by ericnost** [comment]
I am going to go ahead and merge this!

**Pull request merged**